frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -69,12 +69,12 @@
   "This object has been updated.": "This object has been updated.",
   "Click reload to see the new version.": "Click reload to see the new version.",
   "Save": "Save",
+  "Click on the names to access advanced options for ": "Click on the names to access advanced options for ",
   "Triggers": "Triggers",
   "Secrets": "Secrets",
   "Run Policy": "Run Policy",
   "Hooks": "Hooks",
   "Advanced options": "Advanced options",
-  "Click on the names to access advanced options for <1></1>.": "Click on the names to access advanced options for <1></1>.",
```

Comment:
Please let us keep this translation and don't add a dot somewhere else in the code. Not all languages use a dot at the end of a sentence.